project-base/storefront/components/Layout/Header/AutocompleteSearch/AutocompleteSearchPopup.tsx

```diff
@@ -83,12 +83,15 @@ export const AutocompleteSearchPopup: FC<AutocompleteProps> = ({
         callback();
     };
 
+    const shouldDisplaySearchCounts = productsSearch.totalCount !== -1;
```

Comment:
I am not sure if I understand. This constant is based on the total count of products, but we use it as a boolean for displaying the number of e.g. searched brands. Cannot it happen that `productsSearch.totalCount === -1`, but we will find 3 brands and will want to show it?

Reply:
Sadly there was no better way to implement it - it allows only integers, so we could not return null for totalCount so storefront would know, that we do not want to show a count of search results, so we use -1.

-1 says, that returned values are not total counts as we are unable to get exact number of found results per entity from Luigi's Box.

Reply:
That does not fully answer my question. I was more asking how is total count of products related to found e.g. brands. You use this same condition for all entities. Is that correct? Why? It is not clear from the code

<img width="543" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/b5461268-3a1f-4dcb-81c0-46ef8e8cf5c9">